### PR TITLE
Fix Document#elementsFromPoint no viewport available test

### LINKED ISSUE #10093: Don't crash when calling `nodes_from_point` without a document element
Right now, the function `Document::nodes_from_point` is called when using `document.elementsFromPoint`.

That function asserts that there should be a document element to run, and crashes if not. It should return an empty vector in that case, preferrably issuing a `debug!()` message.

Code:
 * `components/script/dom/document.rs` (the function `nodes_from_point`).

### PR DESCRIPTION
**Pull request by luthfianto**
Fixes #10093 and improves #10034.

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/10426)

### CODE REVIEW

File: components/script/dom/document.rs [hunk 1]
```diff
@@ -2681,6 +2679,10 @@ impl DocumentMethods for Document {
         let window = window_from_node(self);
         let viewport = window.window_size().unwrap().visible_viewport;
 
+        if !self.browsing_context().is_some() {
```
Comment (rilut) [inline components/script/dom/document.rs]: Is this correct? If it's correct, should I combine this with ` if x < 0.0 || y < 0.0 || x > viewport.width.get() || y > viewport.height.get() {`?

**Comment by nox** [comment]
@rilut Just change that to `if self.browsing_context().is_none()` and I will r=me.

File: components/script/dom/document.rs [hunk 2]
```diff
@@ -1544,9 +1544,7 @@ impl Document {
         self.browsing_context.is_none() || !url_has_network_scheme(&self.url)
     }
 
-    pub fn nodes_from_point(&self, page_point: &Point2D<f32>) -> Vec<UntrustedNodeAddress> {
-        assert!(self.GetDocumentElement().is_some());
-
+     pub fn nodes_from_point(&self, page_point: &Point2D<f32>) -> Vec<UntrustedNodeAddress> {
```
Comment (notriddle) [inline components/script/dom/document.rs]: Indentation fail. Just press backspace once, so the `pub fn` lines up with the previous line's `}`

**Comment by emilio** [comment]
@bors-servo: r-

Please change the indentation :)

**Pull request merged**